src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (invertedRobot.hasOwnProperty(robot[key])) {
```

Comment:
The use of `hasOwnProperty` is correct for checking if a property exists in an object. However, it's generally safer to call `hasOwnProperty` from the `Object.prototype` directly to avoid issues where the object might have an own property by that name. For example: `Object.prototype.hasOwnProperty.call(invertedRobot, robot[key])`.